site/profile/manifests/core/selinux.pp

```diff
@@ -1,6 +1,6 @@
 class profile::core::selinux {
-  if ($facts['os']['selinux']['enabled']) {
-    notify { "os.selinux.enabled == ${facts['os']['selinux']['enabled']}":
+  if fact('os.selinux.enabled') {
+    notify { "os.selinux.enabled == ${fact('os.selinux.enabled')}":
```

Comment:
Notify resource produces a change. If this is for information only, I would make use of the echo type.

Reply:
The notify only triggers if selinux is enabled... and then the host is rebooted with it disabled and the notify will not trigger again.